src/public/css/sign.css

```diff
@@ -0,0 +1,22 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
+
+html, body{
+    height: 100%;
+    width: 100%;
+    font-family: 'Open Sans' sans-serif;
+    /*color: #ffffff;*/
+
+}
+
+.navbar {
+    position: -webkit-sticky; /* Safari */
```

Comment:
this position: sticky is supposed to keep the element at the top of the page - when I checked it wasn't working, if you remove this does it work?